contracts/commands/AaveV3BasicSellCommandV2.sol

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+/// AaveV3BasicSellCommandV2.sol
+
+// Copyright (C) 2023 Oazo Apps Limited
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.8.22;
+
+import { IServiceRegistry } from "../interfaces/IServiceRegistry.sol";
+import { IPool } from "../interfaces/AAVE/IPool.sol";
+import { IPriceOracleGetter } from "../interfaces/AAVE/IPriceOracleGetter.sol";
+import { RatioUtils } from "../libs/RatioUtils.sol";
+import { IPoolAddressesProvider } from "../interfaces/AAVE/IPoolAddressesProvider.sol";
+import { IAccountImplementation } from "../interfaces/IAccountImplementation.sol";
+import { IOperationExecutor, Call } from "../interfaces/IOperationExecutor.sol";
+import { BaseDMACommand, ICommand } from "./BaseDMACommand.sol";
+
+struct BasicSellTriggerData {
+    address positionAddress;
```

Comment:
I'm missing a bit of comments here explaining what each field is, so we can remember in the future. For example `deviation` is quite ambiguous